docker/data-science/notebooks/ch15/0.setup.ipynb

```diff
@@ -0,0 +1,64 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ltr.download import download, extract_tgz\n",
```

Comment:
Notebooks is currently failing to execute:

`ModuleNotFoundError: No module named 'ltr'`

Reply:
I fixed by adding 
```
import sys
sys.path.append("..")
```

But it looks like there's no movie helper / `indexable_movies` function
`ModuleNotFoundError: No module named 'ltr.helpers.movies'`